e2e-tests/psmdb/run

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+export TEST_DIR=$(realpath $(dirname $0))
+export SRC_DIR="$(realpath ${TEST_DIR}/../../)"
+. ${TEST_DIR}/../functions
+
+NAMESPACE="psmdb-${RANDOM}"
+OPERATOR_IMAGE_TO_UPDATE="perconalab/percona-server-mongodb-operator:master"
+PSMDB_IMAGE_TO_UPDATE="${OPERATOR_IMAGE_TO_UPDATE}-mongod4.0"
+BACKUP_IMAGE_TO_UPDATE="${OPERATOR_IMAGE_TO_UPDATE}-backup"
+
+S3_BUCKET="operator-testing"
+S3_SECRET="minio-secret"
+S3_ENDPOINT="http://minio-service:9000/"
+S3_REGION="us-east-1"
+
+CLUSTER=test-cluster
+BACKUP_NAME=''
+
+init() {
+    export PATH="${SRC_DIR}/build/_output/bin":${PATH}
+
+    create_namespace "${NAMESPACE}"
+    deploy_cert_manager
```

Comment:
sorry, please don't install cert_manager.
user should make minimal actions for cli tool

Reply:
ok